tests/Controller/GluggiControllerTest.php

```diff
@@ -36,36 +36,70 @@ public function testType ()
     public function testComponent ()
     {
         $client = static::createClient();
-        $client->request("GET", "/atom/correct/");
+        $client->request("GET", "/atom/example/");
         $content = $client->getResponse()->getContent();
 
         self::assertContains(
-            "<title>Gluggi // Atom // Correct</title>",
+            "<title>Gluggi // Atom // Example</title>",
             $content
         );
     }
 
 
 
-    public function testGluggiWrapperOnAtom ()
+    public function dataProviderComponentTypes ()
     {
-        $client = static::createClient();
-        $crawler = $client->request("GET", "/atom/correct/");
-
-        // the page should have a header & preview container
-        self::assertSame(1, count($crawler->filter(".gluggi-header")));
-        self::assertSame(1, count($crawler->filter(".gluggi-preview")));
+        // mapping component type on "should have staged view"
+        return [
+            ["atom", true],
+            ["molecule", true],
+            ["organism", true],
+            ["template", true],
+            ["page", false],
+        ];
```

Comment:
Nice catch! 🎈  

Currently there were error 500s if an unknown component type was used (which I just catched with the tests).

This is now fixed, tests are added.